src/routes/game/components/elements/cardPortal/CardPortal.module.css

```diff
@@ -1,7 +1,10 @@
-.popUp {
+.popUpContainer {
```

Comment:
This is the change that fixed the styling issues that you raised @LaustinSpayce. This new class is used on line 65 of the CardPortal.tsx